apps/examples/testcase/ta_tc/wifi_manager/itc/itc_wifi_manager_main.c

```diff
@@ -64,19 +64,19 @@ static wifi_manager_cb_s wifi_callbacks = {
 	wifi_scan_ap_done_cb, // this callback function is called when scanning ap is done.
 };
 
-static void wifi_sta_connected_cb(wifi_manager_result_e res)
+void wifi_sta_connected_cb(wifi_manager_result_e res)
 {
 	printf("wifi_sta_connected: send signal!!! \n");
 	WIFITEST_SIGNAL;
 }
 
-static void wifi_sta_disconnected_cb(void)
+void wifi_sta_disconnected_cb(void)
 {
 	printf("wifi_sta_disconnected: send signal!!! \n");
 	WIFITEST_SIGNAL;
 }
 
-static void wifi_scan_ap_done_cb(wifi_manager_scan_info_s **scan_info, wifi_manager_scan_result_e res)
+void wifi_scan_ap_done_cb(wifi_manager_scan_info_s **scan_info, wifi_manager_scan_result_e res)
```

Comment:
Hi @manoj11588612,
I have confirmed that all of the issues mentioned above have been resolved.
However, in order to cope with scanning failure that may occur in the (cypress) wifi device driver,
please add WIFITEST_SIGNAL before returning in case of WIFI_SCAN_FAIL.